datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/CapturedDataWriterTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Created on 12.08.15 at 16:31
+ */
+package de.cyface.datacapturing.backend;
+
+import java.util.Arrays;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.ProviderTestCase2;
+
+
+/**
+ * Tests whether captured data is correctly saved to the underlying content provider.
+ * The Integration tests need an Activity to be executed, this they are in an own module.
+ *
+ * (!) Create a custom Instrumented Test config for integration-test module to run this in the IDE !
+ *
+ * @author Klemens Muthmann
+ * @version 1.0.0
+ * @since 1.0.0
+ */
+@RunWith(AndroidJUnit4.class)
+public class CapturedDataWriterTest /*extends ProviderTestCase2<MeasuringPointsContentProvider>*/ {
+    /**
```

Comment:
dead code?

Reply:
Not really. I added the relevant code in a later commit under a different task.